### PR TITLE
AB#39377 linting

### PR DESCRIPTION
**Pull request by lhao03**
Added these rules:
```
    "rules": {
        "react/no-unescaped-entities": 0, // ignores this error: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md // if you think it's important I can enable it
        "react/no-direct-mutation-state": 2, // prevents someone from trying to directly change state without using setState
        "react/jsx-uses-react": 2, // prevents someone from importing react if they aren't using react
        "react/jsx-uses-vars": 2, // prevents someone from declaring unused vars
        "semi": [
            2,
            "always" // semicolons
        ]
    }
```
These are all the recommended react and eslint rules.
```
    "extends": [
        "react-app",
        "react-app/jest",
        "eslint:recommended",
        "plugin:react-hooks/recommended", // warns if you use hook incorrectly 
        "plugin:react/recommended"
    ],
```

Added `--fix` to auto fix things like semicolons.

image
Was able to catch this!

### CODE REVIEW

File: tabs/package.json [hunk 1]
```diff
@@ -28,12 +28,13 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "eject": "react-scripts eject",
-    "lint": "eslint src/**/*.js src/**/*.jsx"
+    "lint": "eslint --fix src/*.jsx src/components/sample/*.jsx src/components/sample/lib/*.js"
```
Comment (acchiang) [inline tabs/package.json:31]: shoutout to the --fix flag <3

**Review by acchiang — APPROVED** [review]
LGTM! Should we also implement a Husky pre-commit hook to lint? Just wondering

**Review by gracet9182 — APPROVED** [review]
Nice! Pulled and ran :D

**Review by omersayshi — APPROVED** [review]

**Comment by lhao03** [comment]
> LGTM! Should we also implement a Husky pre-commit hook to lint? Just wondering

Yeah I can look into that!

**Pull request merged**